frontend/ui/widget/sortwidget.lua

```diff
@@ -98,7 +98,9 @@ function SortItemWidget:onTap(_, ges)
             self.item:callback()
         end
     elseif self.show_parent.sort_disabled then
-        return true
+        if self.item.callback then
+            self.item:callback()
+        end
```

Comment:
Sure about removing this `return true` ? (when no item.callback, you would have self.show_parent:_populateItems() run, unlike before.

Reply:
Added `return true` in the `else` branch.